torchvision/models/vision_transformer.py

```diff
@@ -26,6 +26,14 @@
 }
 
 
+class ConvStemLayer(NamedTuple):
```

Comment:
This is not a layer but rather the configuration object of the layer. We got [similar objects](https://github.com/pytorch/vision/blob/eac3dc7bab436725b0ba65e556d3a6ffd43c24e1/torchvision/models/mobilenetv3.py#L38) in vision, so I recommend renaming:
```suggestion
class ConvStemConfig(NamedTuple):
```